### DISCUSSION TITLE
Itineraries?

### DISCUSSION THREAD

**Discussion by rachelpr** [rachelpr/LeafSmart#95] [Ideas]
Would we ever want people to have the ability to build itineraries/things to do in a city?

**Comment by b1u3too** [comment]
Tripadvisor's api may be helpful on this front! [link](https://www.tripadvisor.com/developers)

**Comment by rachelpr** [comment]
I've looked into it a bit but wasn't sure if you could get their infos if you're not a partner! Will have to dig in ⛏️
